### PR TITLE
[Feature] Initialized schema + setup prisma

### PR DESCRIPTION
**Pull request by j-bunk**
Set up initial schema

### CODE REVIEW

**Review by EmilioMena — APPROVED** [review]
Looks good! Great job team 🚀

**Review by chrischan325 — APPROVED** [review]
This is awesome!! You guys are awesome!! :)

File: prisma/schema.sql [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+  mobility_aid_required    BOOLEAN NOT NULL DEFAULT false,
+  cannot_walk_100m         BOOLEAN NOT NULL DEFAULT false,
+  notes                    TEXT,
+  certification_date       DATE,
+  aid                      Aid ARRAY,
+  physician_id             INTEGER NOT NULL,
+  applicant_id             INTEGER UNIQUE NOT NULL,
+  created_at               TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  updated_at               TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP,
+
```
Comment (OustanDing) [inline prisma/schema.sql:118]: Wait did these newlines not affect the execution of the script?

File: prisma/schema.sql [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+  id                SERIAL PRIMARY KEY NOT NULL,
+  rcd_permit_id     INTEGER NOT NULL UNIQUE,
+  expiry_date       DATE NOT NULL,
+  receipt_id        INTEGER,
+  active            BOOLEAN NOT NULL DEFAULT true,
+  application_id    INTEGER NOT NULL,
+  applicant_id      INTEGER NOT NULL,
+  created_at        TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  updated_at        TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP,
+
```
Comment (OustanDing) [inline prisma/schema.sql:190]: Same here

**Review by OustanDing — COMMENTED** [review]
We likely want to get rid of the 2 line breaks I pointed out below. If we do, we should create a backup of `schema.prisma` and before we regenerate it to verify that our custom mappings persist (they should).

File: prisma/schema.sql (hunk 1 above)
Comment (j-bunk) [reply prisma/schema.sql:118]: I just tried out the SQL script to create a new db from scratch and it worked so these line breaks aren't an issue

**Review by OustanDing — APPROVED** [review]
LGTM! Great work team :))

**Pull request merged**